connector/saml/saml.go

```diff
@@ -106,6 +104,9 @@ type Config struct {
 	//		urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
 	//
 	NameIDPolicyFormat string `json:"nameIDPolicyFormat"`
+
+	// Allow a clock_drift when checking dates
+	AllowedClockDrift int64 `json:"allowedClockDrift"`
```

Comment:
Hmmm, why would this be configurable? In other packages we just choose a somewhat arbitrary value and role with it.